src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import core.lottery.Lottery;
+    
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            System.out.println("Got " + lottery.getRandomBall().toString());
```

Comment:
```suggestion
            System.out.println("Got " + lottery.getRandomBall());
```
toString() is called automatically in System.out.println

Reply:
i took away redundant